app/views/common/SPKeyboardAvoidingView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { KeyboardAvoidingView, Platform, StyleSheet } from 'react-native';
+
+export const SPKeyboardAvoidingView = ({ behavior, children }) => {
```

Comment:
could we just name this `KeyboardAvoidingView` ?  We can import the RN version as `import {KeyboardAvoidingView  as RNKeyboardAvoidingView, ... }`

We aren't using an `SP` prefix for any other genric component wrappers (e.g. `<Button />, <Switch />`, etc.), so I feel like it adds a bit of confusion.